src/gebt_poc/point.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "src/rigid_pendulum_poc/vector.h"
+
+using Vector = openturbine::rigid_pendulum::Vector;
```

Comment:
Don't use `using` statements in the global namespace in headers.  Every file that includes this will now have this alias. Put this statement inside the point class itself to encapsulate it.